service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -81,26 +82,32 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         return respond(caseDetails);
     }
 
-    private void run3132(CaseDetails caseDetails) {
+    private void run164(CaseDetails caseDetails) {
         CaseData caseData = getCaseData(caseDetails);
-        Long id = caseData.getId();
-        State state = caseData.getState();
-        if (IGNORED_STATES.contains(state)) {
-            throw new AssertionError(format(
-                "Migration {id = FPLA-3132, case reference = %s} not migrating when state = %s", id, state
-            ));
-        }
-        int numChildren = caseData.getAllChildren().size();
-        if (MAX_CHILDREN < numChildren) {
+
+        if (caseData.getId() != 1626258358022834L) {
             throw new AssertionError(format(
-                "Migration {id = FPLA-3132, case reference = %s} not migrating when number of children = %d (max = %d)",
-                id, numChildren, MAX_CHILDREN
+                "Migration {id = DFPL-164, case reference = %s}, expected case id 1626258358022834",
+                caseData.getId()
             ));
         }
 
-        caseDetails.getData().putAll(populator.generate(caseData, CHILD, BLANK));
+        List<Element<CourtAdminDocument>> otherCourtAdminDocuments = caseData.getOtherCourtAdminDocuments();
+
+        Element<CourtAdminDocument> documentElement = otherCourtAdminDocuments.stream()
+            .filter(documents ->
+                "LA Certificate.pdf".equals(documents.getValue().getDocument().getFilename()))
+            .findFirst()
+            .orElseThrow();
+
+        log.info("Certificate document found");
```

Comment:
whilst not required it might be nice to prefix these logs like you have with the assertion error
```java
log.info("Migration {id = DFPL-164, case reference = {}} Certificate document found", caseData.getId())
```
and the same for the log below, if you do do this then it would be sensible to store the id as a var at the start of the method.

Reply:
Changes done